config/covid-19/test/contact-summary/contact-summary.spec.js

```diff
@@ -0,0 +1,99 @@
+const { expect } = require('chai');
+const TestRunner = require('medic-conf-test-harness');
+
+const harness = new TestRunner();
+
+describe('Contact Summary', () => {
+  before(async () => await harness.start());
+
+  after(async () => await harness.stop());
+
+  beforeEach(async () => {
+    await harness.clear();
+    await harness.setNow('2021-01-01');
+  });
+
+  afterEach(() => {
+    expect(harness.consoleErrors).to.be.empty;
+  });
+
+  it('should return context and cards empty', async () => {
+    const contact = { name: 'contact', type: 'person' };
+    const reports = [];
+    const lineage = [{ contact }];
+    const summary = await harness.getContactSummary(contact, reports, lineage);
+
+    expect(summary.context).to.deep.equal({});
+    expect(summary.cards).to.not.be.undefined;
+    expect(summary.cards.length).to.equal(0);
+  });
+
+  it('should return fields with right values for person contact', async () => {
+    const contact = {
```

Comment:
Can use a [factory ](https://github.com/rosiejs/rosie) object for data. 

```
const personFactory = require('../../../../factories/cht/contacts/person'); // I think I have the path right.

personFactory.build();
```
and if the default data doesn't meet your needs. You can override as needed or maybe update to be a globally better. 

`personFactory.build({address: '1 Will St Emer town'});`


https://github.com/medic/cht-core/blob/master/tests/factories/cht/contacts/person.js

Reply:
Very nice! 
I didn't used the one from `require('../../../../factories/cht/contacts/person')` because that creates a dependency between core's src files and configurations. I consider these configurations to be able to run test independently of it's location or regardless if the core's src is in the same repo (production repos doesn't have core files in it).
Instead I added `rosie` lib and created factory files that set fields as it's defined in the config and use them in the tests, hopefully this is a nice sample for other configs to do the same as it's so easy.